Marlin/src/pins/pins_MKS_SBASE.h

```diff
@@ -0,0 +1,295 @@
+/**
+ * Marlin 3D Printer Firmware
+ * Copyright (C) 2016 MarlinFirmware [https://github.com/MarlinFirmware/Marlin]
+ * Copyright (C) 2017 MarlinFirmware [https://github.com/MarlinFirmware/Marlin]
+ *
+ * Based on Sprinter and grbl.
+ * Copyright (C) 2011 Camiel Gubbels / Erik van der Zalm
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+/**
+ * MKS SBASE pin assignments
+ */
+
+//#if !defined(TARGET_LPC1768)
+#if DISABLED(IS_REARM)
+  #error "Oops!  Make sure you have Re-Arm selected."
+#endif
+
+#ifndef BOARD_NAME
+  #define BOARD_NAME "MKS SBASE"
+#endif
+
+#define LARGE_FLASH true
```

Comment:
Add here also:
```cpp
#define DEFAULT_WEBSITE_URL "https://www.makerbase.com.cn"
```
…or maybe…
```cpp
#define DEFAULT_WEBSITE_URL "https://github.com/makerbase-mks/MKS-SBASE"
```

Reply:
@thinkyhead 
config/examples/Makerbase/SBASE/ , this was ment for the dev''s not me ?
Will put inn the DEFAULT_WEBSITE_URL at the same time i check over the LCD pin's :)

+//#if !defined(TARGET_LPC1768)
+#if DISABLED(IS_REARM)
+  #error "Oops!  Make sure you have Re-Arm selected."

I was told to leave it like this i think, it will break the platformio.ini if i change it ?